cmd/image-builder/main.go

```diff
@@ -213,15 +212,10 @@ func cmdBuild(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	var uploadUnsupported *UploadTypeUnsupportedError
-	var missingUploadConfig *MissingUploadConfigError
 	uploader, err := uploaderFor(cmd, res.ImgType.Name())
-	if err != nil && !errors.As(err, &missingUploadConfig) && !errors.As(err, &uploadUnsupported) {
+	if err != nil {
```

Comment:
The reason we have the error types was to distinguish between the following cases:
- image type cannot be uploaded, ignore errors from creating an uploader in this case
- no upload config provided: the user has no intend to upload, ignore the error in this case
- incomplete upload config: the user seems to want to upload, so we need to error to avoid surprises

This still needs to be handled here.